test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -590,44 +590,34 @@ private static void clearClusters() throws Exception {
     }
 
     private void afterInternal(boolean afterClass) throws Exception {
-        boolean success = false;
```

Comment:
Just FYI for reviewers, the diff here is very difficult to read unless you enable the "hide whitespace" setting:

<img width="239" alt="image" src="https://github.com/opensearch-project/OpenSearch/assets/9160839/ecae57a5-52ea-4b86-9b94-330ad665837c">